generators/_init-hybrid/templates/plain/.circleci/config.yml

```diff
@@ -80,17 +143,49 @@ workflows:
 #                - develop
 #    jobs:
 #      - build
```

Comment:
Also update the build steps here. So that the workflow still works when uncommenting.